# name: curses.error 
exception curses.error